docker/structure.sql

```diff
@@ -289,3 +289,20 @@ CREATE TRIGGER hog_tasks_moddatetime
     EXECUTE PROCEDURE moddatetime (updated_at);
 
 CREATE INDEX idx_task_coalition ON hog_tasks(coalition);
+
+
+CREATE TABLE artifacts (
+    id SERIAL PRIMARY KEY,
+    type text,
+    key text,
+    data json,
```

Comment:
As mentioned in your comment this could probably be a text. Should make things a little sleeker